subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c

```diff
@@ -538,6 +543,22 @@ size_t cracen_platform_keys_get_size(psa_key_attributes_t const *attributes)
 	return 0;
 }
 
+bool cracen_platform_keys_is_ikg_key(psa_key_attributes_t const *attributes)
+{
+	platform_key key;
+	key_type type = find_key(MBEDTLS_SVC_KEY_ID_GET_KEY_ID(psa_get_key_id(attributes)), &key);
+	if (type == IKG) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+uint32_t cracen_platform_keys_get_owner(psa_key_attributes_t const *attributes)
+{
+	return PLATFORM_KEY_GET_DOMAIN(MBEDTLS_SVC_KEY_ID_GET_KEY_ID(psa_get_key_id(attributes)));
```

Comment:
(minor) Owner or domain? Could we standardize on a single naming?

Reply:
I don't think that this is wrong. So the key itself has an owner, which for the platform keys is the domain. The key owner is a more generic term and in the PSA context can refer to different things. For example in TF-M the key owner is the TF-M partitions which owns the key.